ui/imports/shared/controls/TransactionDelegate.qml

```diff
@@ -1,148 +1,94 @@
-                transactionListItem.symbol = token.symbol
-                break
-            }
-        }
-    }
 
-    MouseArea {
-        anchors.fill: parent
-        onClicked: launchTransactionModal()
-        cursorShape: Qt.PointingHandCursor
-        hoverEnabled: true
-        onEntered: {
-            transactionListItem.isHovered = true
-        }
-        onExited: {
-            transactionListItem.isHovered = false
+StatusListItem {
+    id: root
+
+    property var modelData
+    property string symbol
+    property bool isIncoming
+    property int transferStatus
+    property string currentCurreny
+    property string cryptoValue
+    property string fiatValue
+    property string networkIcon
+    property string networkColor
+    property string networkName
+    property string shortTimeStamp
+    property string resolvedSymbol: root.symbol != "" ? root.symbol : "ETH"
+    property string savedAddressName
+
+    image.source: Style.png("tokens/%1".arg(resolvedSymbol))
+    statusListItemTitle.font.weight: Font.Medium
+    title: isIncoming ? qsTr("Receive %1").arg(resolvedSymbol) : !!savedAddressName ?
+                            qsTr("Send %1 to %2").arg(resolvedSymbol).arg(savedAddressName) :
+                            qsTr("Send %1 to %2").arg(resolvedSymbol).arg(Utils.compactAddress(modelData.to, 4))
+    subTitle: shortTimeStamp
+    inlineTagModel: 1
+    inlineTagDelegate: InformationTag {
+        tagPrimaryLabel.text: networkName
+        tagPrimaryLabel.color: networkColor
+        image.source: Style.svg("tiny/%1".arg(networkIcon))
+        background: Rectangle {
+            id: controlBackground
+            implicitWidth: 51
+            implicitHeight: 24
+            color: "transparent"
+            border.width: 1
+            border.color: Theme.palette.baseColor2
+            radius: 36
         }
+        rightComponent: transferStatus === 1 ? completedIcon : loadingIndicator
     }
-
-    Row {
-        anchors.left: parent.left
-        anchors.leftMargin: Style.current.smallPadding
-        anchors.verticalCenter: parent.verticalCenter
-        spacing: 5
-
-        Image {
-            id: assetIcon
-            width: 40
-            height: 40
-            source: Style.png("tokens/"
-                    + (transactionListItem.symbol
-                       != "" ? transactionListItem.symbol : "ETH"))
-            anchors.verticalCenter: parent.verticalCenter
-            onStatusChanged: {
-                if (assetIcon.status == Image.Error) {
-                    assetIcon.source = Style.png("tokens/DEFAULT-TOKEN@3x")
+    components: [
+        ColumnLayout {
+            Row {
+                Layout.alignment: Qt.AlignRight
+                spacing: 4
+                StatusIcon {
+                    color: isIncoming ? Theme.palette.successColor1 : Theme.palette.dangerColor1
+                    icon: "arrow-up"
+                    rotation: isIncoming ? 135 : 45
```

Comment:
We have distinct arrow icons for up/down

Reply:
<img width="651" alt="image" src="https://user-images.githubusercontent.com/60327365/189095287-1e46a448-2bd6-4fb0-9297-6cad4bacc5dd.png">
This is the what I need on this case!